vmngclient/endpoints/__init__.py

```diff
@@ -135,6 +136,17 @@ def asdict(self) -> Dict[str, Any]:
         return result
 
 
+def dict_transform_enum_values_to_str(kwargs: Dict[str, Any]) -> Dict[str, str]:
+    # this is to keep compatiblity and have seme behavior for (str, Enum) mixin after 3.11 for url formatting
+    result: Dict[str, str] = {}
+    for arg, val in kwargs.items():
+        if isinstance(val, Enum):
+            result[arg] = str(val.value)
+        else:
+            result[arg] = val
+    return result
+
+
```

Comment:
From function definition, we do not know if `result` will have always `str` as a value. If we know that `value` is always sublass of `str`, maybe we should always cast `value` to `str`? 

This won't break typing rules and if anything would go wrong, we can catch an exception here.

Reply:
fixed